frontend/styles/components/calendar/DailyViewRow.module.scss

```
@@ -1,9 +1,26 @@
+@import '../../Variables.module.scss';
```

Stylelint (17.14.1)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)